src/main/cpp/OI.cpp

```diff
@@ -132,21 +134,22 @@ OI::OI()
   dStart.WhenPressed(new StageOneClimb());
   dA.WhenPressed(new StageTwoClimb());
   dX.WhenPressed(new StageThreeClimb());
-  dY.WhenPressed(new ClimberEngage);
-  dB.WhenPressed(new ClimberDisengage);
+  dY.WhenPressed(new ClimberEngage());
+  dB.WhenPressed(new ClimberDisengage());
   //dY.WhenPressed(new StageFiveClimb());
   //dB.WhenPressed(new StageFiveClimb());
+  dLB.WhenPressed(new dLBPressed());
   dLB.WhenReleased(new dLBReleased());
   // dStart.WhenPressed(new SetVisionDrive());
-  dBack.WhenPressed(new SetControlDrive());
+  dBack.WhenPressed(new FullResetDrive());
 
   coLB.WhenPressed(new HatchIntakeToggle());
   
   coStart.WhenPressed(new FullElevatorLevel(kElevatorCargoLevel_Ship));
 
   coX.WhileHeld(new FullCargoIntake());
 
-  coPOV0.WhenPressed(new FullArmPosition(-21.));
+  coPOV0.WhenPressed(new FullArmPosition(21)); //-21 for new HatchIntake 
```

Comment:
The comment says for it to be -21 but we ended up using 21. We should get rid of this comment to avoid confusion but it's not a major change we have to do.